pydicom/tests/test_values.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2008-2017 pydicom authors. See LICENSE file for details.
+"""Tests for dataset.py"""
+
+import pytest
+
+from pydicom.tag import Tag
+from pydicom.values import (convert_value, converters, convert_tag,
+                            convert_ATvalue, convert_DA_string)
+
+
+def assert_raises_regex(type_error, message, func, *args, **kwargs):
+    """Test a raised exception against an expected exception.
+
+    Parameters
+    ----------
+    type_error : Exception
+        The expected raised exception.
+    message : str
+        A string that will be used as a regex pattern to match against the
+        actual exception message. If using the actual expected message don't
+        forget to escape any regex special characters like '|', '(', ')', etc.
+    func : callable
+        The function that is expected to raise the exception.
+    args
+        The callable function `func`'s arguments.
+    kwargs
+        The callable function `func`'s keyword arguments.
+
+    Notes
+    -----
+    Taken from https://github.com/glemaitre/specio, BSD 3 license.
+    """
+    with pytest.raises(type_error) as excinfo:
+        func(*args, **kwargs)
+    excinfo.match(message)
+
+
+class TestConvertTag(object):
+    def test_big_endian(self):
+        """Test convert_tag with a big endian byte string"""
+        bytestring = b'\x00\x10\x00\x20'
+        assert convert_tag(bytestring, False) == Tag(0x0010, 0x0020)
+
+    def test_little_endian(self):
+        """Test convert_tag with a little endian byte string"""
+        bytestring = b'\x10\x00\x20\x00'
+        assert convert_tag(bytestring, True) == Tag(0x0010, 0x0020)
+
+    def test_offset(self):
+        """Test convert_tag with an offset"""
+        bytestring = b'\x12\x23\x10\x00\x20\x00\x34\x45'
+        assert convert_tag(bytestring, True, 0) == Tag(0x2312, 0x0010)
+        assert convert_tag(bytestring, True, 2) == Tag(0x0010, 0x0020)
+
+    @pytest.mark.skip(reason='empty bytestring not handled properly')
+    def test_empty_bytestring(self):
+        """Test convert_tag with empty bytestring"""
+        bytestring = b''
+        assert convert_tag(bytestring, True) == ''
+
+    @pytest.mark.skip(reason='bad bytestring not handled properly')
+    def test_bad_bytestring(self):
```

Comment:
Did you write an issue for these ones?
EDIT: Ok, you wrote a TODO later in the code.